src/main/scala/viper/gobra/backend/BackendVerifier.scala

```diff
@@ -49,10 +50,9 @@ object BackendVerifier {
 
     val verificationResults =  {
       val verifier = config.backend.create(exePaths, config)
-      val programID = s"_programID_${config.inputs.map(_.name).mkString("_")}"
 
       if (!config.shouldChop) {
-        verifier.verify(programID, BacktranslatingReporter(config.reporter, task.backtrack, config), task.program)(executor)
+        verifier.verify(task.name, BacktranslatingReporter(config.reporter, task.backtrack, config, task.name), task.program)(executor)
```

Comment:
In my previous comment, I mentioned that you could drop the `programID` because it was derived from the `task.name`. What I meant was actually that you could probably drop one parameter altogether - If for some reason the first and last parameters are always the same, could we drop one of them?

Reply:
I am still not sure what you mean, but I moved `BacktranslatingReporter(config.reporter, task.backtrack, config, task.name)` out of there since it's the same for the normal and the chopper case

Reply:
Oops, my bad, for some reason I did not see that the task.name is a param to the BacktranslatingReporter. Nevermind this comment